js/index.js

```diff
@@ -0,0 +1,210 @@
+'use strict'
+
+const blinding = (iter, on = null) => {
+    const action = on == null ?
+        "toggle" :
+        on ?
+            "add" :
+            "remove"
```

Comment:
```suggestion
    const action = !!on        ? "add"    :
                   on === null ? "toggle" :
                   "remove"
```
- 조금 더 가독성을 고려해보시면 좋을 것 같아요!
- on이 truthy일때는 add이고 falsy일 경우에서 null일때는 toggle입니다.  
- 차라리 순서를 좀 바꿔보면 어떨까요?
  - truthy일 때 add
  - falsy중 null일때 toggle
  - 아니면 remove

Reply:
다른 함수들 처럼 능동형 동사가 되면, 좀 더 구체화를 하려면 목적어를 붙이는게 좀 더 직관적일 것 같은데 어떻게 생각하시나요?

Reply:
@jho2301 제 생각에도 그렇게 하는게 좋을 것 같습니다!